src/java/htsjdk/samtools/metrics/MetricsFile.java

```diff
@@ -427,6 +433,7 @@ else if (line.startsWith(MINOR_HEADER_PREFIX)) {
                         final HKEY key = (HKEY) formatter.parseObject(fields[0], keyClass);
 
                         for (int i=1; i<fields.length; ++i) {
+                            if(fields[i].equals("")) continue;
                             final double value = formatter.parseDouble(fields[i]);
```

Comment:
This was added to allow the code to handle multiple differently sized histograms, for example the first one is size 5, second is size 1 and third is size 5. It needs to skip the empty spots to parse correctly.